src/Elliptic/Executables/Linear/SolveLinearEllipticProblem.hpp

```diff
@@ -53,6 +53,16 @@
 #include "Utilities/Functional.hpp"
 #include "Utilities/TMPL.hpp"
 
+namespace detail {
```

Comment:
what's the reasoning for putting this in a `detail` namespace?

Reply:
you're right, it shouldn't be

Reply:
Actually now I remember why I placed it in a `detail` namespace: The `LinearSolverGroup` is specific to this executable, so to make sure it doesn't collide with anything in `::OptionTags` it needs to be in some namespace. Putting it in a `detail` namespace seemed the easiest thing to do. Any thoughts on this?

Reply:
I think I'd prefer a namespace that is more reminiscent of the executable -- `LinearElliptic` maybe? My reasoning is that `detail` tends to indicate that the code is implementation details and not considered part of the supported interface, but if an option tag alters how the input file should end up, it ought to be considered part of the library interface.

Reply:
As far as I can tell, the actual namespace name appears only three times: twice in the `namespace blah { .. } // namespace blah` lines, and once in the main body of the executable. So the very unambiguous, if cumbersome, `namespace SolveLinearEllipticProblem` might also be an acceptable choice.

Reply:
ok, named `SolveLinearEllipticProblem`